docs/commands/filteralleles.md

```diff
@@ -73,9 +73,13 @@ The following command removes alternate alleles whose allele count is zero and u
 
 ```
 ... \
+  filtersamples list --remove -i samples_to_exclude.txt \
   filteralleles --remove \
     -c 'va.info.AC[aIndex - 1] == 0' \
-    -a 'va.info.AC = aIndices[1:aIndices.length].map(i => va.info.AC[i - 1])'
+    -a 'va.info.AC = va.info.AC = aIndices[1:].map(i => va.info.AC[i - 1]),
+      va.info.AF = aIndices[1:].map(i => va.info.AF[i - 1]),
+      va.info.MLEAC = aIndices[1:].map(i => va.info.MLEAC[i - 1]),
+      va.info.MLEAF = aIndices[1:].map(i => va.info.MLEAF[i - 1])'
 ```
```

Comment:
@cseed @tpoterba I'm really starting to think we should expose variables called `aaIndex` and `aaIndices` because that's what is actually useful to the user.